src/test/java/org/folio/rest/api/LocationsTest.java

```diff
@@ -200,7 +201,7 @@ public void canListLocations()
     CompletableFuture<Response> getCompleted = new CompletableFuture<>();
     send(locationsStorageUrl("/"), HttpMethod.GET,
       null, SUPPORTED_CONTENT_TYPE_JSON_DEF, ResponseHandler.json(getCompleted));
-    Response getResponse = getCompleted.get(5, TimeUnit.SECONDS);
+    Response getResponse = getCompleted.get(TIMEOUT, TimeUnit.SECONDS);
```

Comment:
It is worth introducing a test utility method, e.g.:

``` java
public static <T> T get(CompletableFuture<T> future) throws Exception {
  return future != null ? future.get(10, SECONDS) : null;
}
```

Reply:
I don't think it's worth. The plain API is simple enough. And perhaps the constant might be useful for other waiting things.

Reply:
The requested get method has been added as a test utility method.

Reply:
I don't see a need for the null test. The NullPointerException will be either in get(CompletableFuture) or when calling getResponse.get...:

    Response getResponse = get(getCompleted);
    assertThat(getResponse.getStatusCode(), is(HttpURLConnection.HTTP_OK));